.github/ISSUE_TEMPLATE/enhancement.yml

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Enhancement
+description: Raise a potential enhancement.
+labels: ["enhancement"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please add as much relevant information to the form below as you can.
+        This will help us replicate and solve the issue. Thank you!
```

Comment:
looks like this needs to be updated? I guess here we want to ask as much details about the enhancement / feature request as we can.

Reply:
Can we update this to , "Please add as much relevant information regarding your enhancement to the form below as you can. The enhancement can be a feature , to a request for additional documentation. Thank you !"

Reply:
Updated. Thanks for the suggestions!